website/project/model.py

```diff
@@ -1587,6 +1594,10 @@ def save(self, *args, **kwargs):
         if settings.PIWIK_HOST and update_piwik:
             piwik_tasks.update_node(self._id, saved_fields)
 
+        # Ensure Mailing list subscription is set up
+        if first_save and self.mailing_enabled:
+            auth_signals.node_created.send(self)
```

Comment:
I would expected `node_created` to fire any time a node is created. Since we're only firing it when `mailing_enabled`, it seems that we should rename it to something like `mailable_node_created`.